### PR TITLE
Add SQLAlchemy to the README

### PR DESCRIPTION
**Pull request by meganle**
We added this library and didn't update the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ An open-sourced backend for the Eatery application, available on Android and iOS
 Technologies involved include:
 1. Flask
 2. GraphQL
+3. SQLAlchemy
```
Comment (kevinchan159) [inline README.md:7]: 😁

**Review by kevinchan159 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -46,4 +47,4 @@ If using VS Code, install the 'Python' extension and include following snippet i
 "python.linting.pylintEnabled": false,
 "python.linting.flake8Enabled": true,
 "python.formatting.provider": "black"
- ```
\ No newline at end of file
+ ```
```
Comment (kevinchan159) [inline README.md:50]: Unnecessary, please remove

**Review by kevinchan159 — CHANGES_REQUESTED** [review]

**Review by kevinchan159 — APPROVED** [review]
W O W

**Review by orkosinha — APPROVED** [review]
Bet

**Pull request closed without merging**